merqube_client_lib/api_client/merqube_client.py

```diff
@@ -22,16 +21,22 @@ class MerqubeAPIClient(IndexAPIClient, SecAPIClient):
     """
 
 
-class MerqubeAPIClientSingleIndex(MerqubeApiClientBase):
+class MerqubeAPIClientSingleIndex(MerqubeAPIClient):
     """
     Class that contains methods that deal with a single index
+    Instantiate with an index name ("name" field in the index manifest) and indicate if it is a realtime index
     """
 
-    def __init__(self, index_name: str, *args: Any, **kwargs: Any) -> None:
+    def __init__(self, index_name: str, is_intraday: bool = False, *args: Any, **kwargs: Any) -> None:
         super().__init__(*args, **kwargs)
 
         res = self.session.get_collection_single(f"/index?names={index_name}")
         self.index_id = res["id"]
+        self.is_intraday = is_intraday
```

Comment:
Instead of making the user provide this, can we just inspect the manifest?

```
self.is_intraday = res.get("intraday", {}).get("enabled", False)
```

Something like `has_intraday` might be a better name since so far all the RT indices are also EOD indices.

Reply:
ah didn't know that (about the dual) - thanks will do now